pages/common/hydra.md

```diff
@@ -0,0 +1,28 @@
+# hydra
+
+> Password brute forcing tool.
+> More information: <https://github.com/vanhauser-thc/thc-hydra/>.
+
+- Run hydra against an ftp server:
+
+`hydra -l {{Username}} -P {{Password List}} ftp://{{IP Address}}`
+
+- Run hydra against an imap server with plain authentication:
+
+`hydra -L {{Username List}} -p {{Password}} imap://{{IP Address}}/PLAIN`
+
+- Run hydra against a pop3s server. The credentials file should be a list in the form `user:password`:
+
+`hydra -C {{Credentials File}} -6 pop3s://[{{IP v6 Address}}]:{{Port}}/TLS:DIGEST-MD5`
+
+- Run hydra against all ftp servers in an ip range, testing a specific username and password:
```

Comment:
Let us remove the word "hydra" from here and henceforth. It is implicit after 3 times.